.gitignore

```diff
@@ -0,0 +1,5 @@
+/.idea/
```

Comment:
It's in [ezplatform](https://github.com/ezsystems/ezplatform/blob/master/.gitignore) repo too.